src/main/java/seedu/duke/command/ListNoteCommand.java

```diff
@@ -18,21 +22,16 @@
 
     public static final String COMMAND_WORD = "list-n";
 
-    private static final String COMMAND_USAGE = COMMAND_WORD + ": Lists all the notes in the Notebook. Parameters: "
+    public static final String COMMAND_USAGE = COMMAND_WORD + ": Lists all the notes in the Notebook. Parameters: "
             + "[" + PREFIX_DELIMITER + PREFIX_TAG + " TAG "
             + PREFIX_DELIMITER + PREFIX_TAG + " TAG1...] "
             + "[up/down]";
 
-    /**
-     * Gets how the command is expected to be used.
-     *
-     * @return String representation of how the command is to be used.
-     */
-    public static String getCommandUsage() {
-        return COMMAND_USAGE;
-    }
+    public static final String COMMAND_MESSAGE_UNSUCCESSFUL = "Your tags return no result."
```

Comment:
This can be private since its only used within this class.

Reply:
You are right, my bad! Thanks